amazonia/classes/autoscaling_unit.py

```diff
@@ -54,10 +55,11 @@ def __init__(self, unit_title, vpc, template, protocols, public_cidr, instancepo
             instanceports=instanceports,
             loadbalancerports=self.loadbalancerports,
             path2ping=path2ping,
-            subnets=public_subnets,
+            subnets=public_subnets if public_unit is True else private_subnets,
```

Comment:
I don't think you need the "is True" when you are using a boolean (you don't need to evaluate it)